docs/docs/cmd/entra/enterpriseapp/enterpriseapp-remove.mdx

```diff
@@ -0,0 +1,71 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# entra enterpriseapp remove
+
+Deletes an enterprise application (or service principal)
+
+## Usage
+
+```sh
+m365 entra enterpriseapp remove [options]
+```
+
+## Alias
+
+```sh
+m365 aad sp remove [options]
+m365 entra sp remove [options]
+```
+
+## Options
+
+```md definition-list
+`-i, --appId [appId]`
+: ID of the application for which the enterprise application should be deleted.
+
+`-n, --appDisplayName [appDisplayName]`
+: Display name of the application for which the enterprise application should be deleted.
+
+`--appObjectId [appObjectId]`
+: ObjectId of the application for which the enterprise application should be deleted.
```

Comment:
I don't really like the fact that we're repeating every time what the command does (`for which the enterprise application should be deleted`). To me it's quite obvious since the command description tells this already.
Let's try to rephrase it a bit without verbose info.